doc/source/examples/quickstart.ipynb

```diff
@@ -1,28 +1,29 @@
 {
```

Comment:
<p>Would it be a good idea to append with <code>;</code> to suppress the memory output? Would the picture still render? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MDAnalysis/UserGuide/pull/241/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='70'/>

Reply:
<p>works! It doesn't make the tests fail, but it does give a nicer output.</p>